### DISCUSSION TITLE
Correct measurement unit?

### DISCUSSION THREAD

**Discussion by walteriviera** [mlco2/codecarbon#582] [Q&A] [answered]
I used the offline emissions tracker and now I have access to the csv file with the cpu power, ram power, and emissions.
Based on the methodology described in your documentation, I should multiply C (carbon) * E (energy) to get the CO2_eq.

Now the C seems to be represented in "g" of CO2, while the Energy is kW/h. However, in the provided example, the measurement is in KgCO2/KWh. This might has to do with the table values expressed in Kg/MWh, but is not clear how we go from there to KgCO2/KWh.

If I take the csv values of C and E and I multiply them as they are, what am I getting? g/KWh? or Kg/KWh?
Thanks

**Comment by benoit-cty** [comment, marked as answer]
What do you mean by "get the CO2_eq" ? CodeCarbon already does the operation "carbon intensity in g/kWh * kWh", that's what you get in "emission".
The emissions are in Kg.eq.CO2.
